docs/sources/administration/correlations/create-a-new-correlation/index.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Create a new correlation
+weight: 40
+---
+
+# Create a new correlation
+
+## Before you begin
+
+Make sure you have permission to add new correlations. Only users with write permissions to data sources can define new correlations.
+
+## Create a correlation in Administration page
+
+1. Go to the Administration section in Grafana
+1. Open Correlations page
+1. Click the “Add” button in the top right corner
+1. A wizard will guide you through the correlation setup
+1. Provide a **label** for the correlation.
+1. Provide an optional **description**
+1. Go to the next page.
+1. Provide **target data source**
+1. Provide **target query** using variables
+1. Go to the next page
+1. Provide **source data source**
+1. Provide **results field**
+1. Add transformations if you need variables that are not fields in the source data source.
+1. Click “Add” to add a new transformation
+1. Select the type of a transformation
+1. Configure transformation depending on the selected type
+1. Save correlation
```

Comment:
some of the items in the list have a period at the end, some don't, maybe we should be more consistent: https://grafana.com/docs/writers-toolkit/style-guide/style-conventions/#unordered-lists

Reply:
Thanks, punctuation updated in 42d831d39c95c1c8301779d93385d211729e8f5e